proto/forch_configuration.proto

```diff
@@ -88,6 +88,9 @@ message OrchestrationConfig {
   // authentication configurations
   AuthConfig auth_config = 7;
 
+  // orchestrated testing configurations
+  OrchestratedTestingConfig orch_testing_config = 11;
```

Comment:
I think just fot_config  -- trying to propagate orch_testing everywhere will be very verbose.  Normally I don't like abbreviations, but I think in this case it works better.  We're starting to call it "Faucet Orchestrated Testing" just for this reason (because OT is a bad abbreviation, FOT is better).

Reply:
updated